sycl/source/backend/opencl.cpp

```diff
@@ -44,6 +46,66 @@ __SYCL_EXPORT queue make_queue(const context &Context,
   return detail::make_queue(NativeHandle, Context, nullptr, false,
                             ContextImpl->get_async_handler(), backend::opencl);
 }
+
+// Free functions to query OpenCL backend extensions
+// TODO: Extensions have been deprecated for aspects
+__SYCL_EXPORT bool has_extension(const sycl::platform &SyclPlatform,
+                                 const std::string &Extension) {
+  if (SyclPlatform.get_backend() != sycl::backend::opencl) {
+    throw sycl::exception(
+        errc::backend_mismatch,
+        "has_extension can only be used with an OpenCL backend");
+  }
+
+  auto PlatformImpl = getSyclObjImpl(SyclPlatform);
+  RT::PiPlatform PluginPlatform = PlatformImpl->getHandleRef();
+  plugin Plugin = PlatformImpl->getPlugin();
```

Comment:
```suggestion
  plugin &Plugin = PlatformImpl->getPlugin();
```

Reply:
This required adding a `const` qualifier, so did this change and that in both methods.